webpack.config.js

```diff
@@ -0,0 +1,58 @@
+const webpack = require('webpack');
+const path = require('path');
+const nodeExternals = require('webpack-node-externals');
+
+const common = {
+  entry: './src/index.ts',
+  mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.ts?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.ts', '.js'],
+  },
+};
+
+const serverConfig = {
```

Comment:
why webpack for node? Is publishing the output from `tsc` directly like we do for the SDK not enough here?

Reply:
thanks! it's updated, we're using webpack just to get the client output